drivers/dma/dmamux_stm32.c

```diff
@@ -53,33 +53,33 @@ struct dmamux_stm32_config {
 };
 
 /*
- * UTIL_LISTIFY is used to generate arrays with function pointers to check
+ * LISTIFY is used to generate arrays with function pointers to check
  * and clear interrupt flags using LL functions
  */
-#define DMAMUX_CHANNEL(i, _)		LL_DMAMUX_CHANNEL_ ## i,
-#define IS_ACTIVE_FLAG_SOX(i, _)	LL_DMAMUX_IsActiveFlag_SO  ## i,
-#define CLEAR_FLAG_SOX(i, _)		LL_DMAMUX_ClearFlag_SO ## i,
-#define IS_ACTIVE_FLAG_RGOX(i, _)	LL_DMAMUX_IsActiveFlag_RGO  ## i,
-#define CLEAR_FLAG_RGOX(i, _)		LL_DMAMUX_ClearFlag_RGO ## i,
+#define DMAMUX_CHANNEL(i, _)		LL_DMAMUX_CHANNEL_ ## i
+#define IS_ACTIVE_FLAG_SOX(i, _)	LL_DMAMUX_IsActiveFlag_SO  ## i
+#define CLEAR_FLAG_SOX(i, _)		LL_DMAMUX_ClearFlag_SO ## i
+#define IS_ACTIVE_FLAG_RGOX(i, _)	LL_DMAMUX_IsActiveFlag_RGO  ## i
+#define CLEAR_FLAG_RGOX(i, _)		LL_DMAMUX_ClearFlag_RGO ## i
 
 uint32_t table_ll_channel[] = {
-	UTIL_LISTIFY(DT_INST_PROP(0, dma_channels), DMAMUX_CHANNEL)
+	LISTIFY(DT_INST_PROP(0, dma_channels), DMAMUX_CHANNEL)
```

Comment:
Does this also need a separator?

Reply:
fixed. let's wait for green CI which should catch most of conversion errors.